<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Required meta tags always come first -->
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  
    <title>Accepted Papers | ICML 2025 Workshop on Multi-modal Foundation Models and Large Language Models for Life Sciences</title>
    <link rel="canonical" href="../pages/speakers.html">
  
    <!-- Custom Fonts and Styles for Improved Aesthetics -->
    <link rel="stylesheet" href="../theme/css/bootstrap.min.css">
    <link rel="stylesheet" href="../theme/css/font-awesome.min.css">
    <link rel="stylesheet" href="../theme/css/pygments/default.min.css">
    <link rel="stylesheet" href="../theme/css/style.css">
    <link rel="stylesheet" href="../theme/css/custom.css">
  
    <style>
        body {
      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
      line-height: 1.5;
      color: #333;
      background-color: #fff;
      margin: 0;
      padding: 0;
    }
        header {
            text-align: center;
            padding: 20px 0;
            background-color: #fff;
            color: #222;
            border-bottom: 1px solid #ddd;
        }
        header h1 {
            font-weight: 600;
            margin: 0;
            font-size: 24px;
        }
        header p {
            font-size: 16px;
            margin-top: 5px;
            color: #666;
        }
        h1, h2, h3, h4, h5, h6 {
      margin-top: 0.5em;
      margin-bottom: 0.5em;
      font-weight: 600;
      line-height: 1.25;
    }

    h1 { font-size: 2.25rem; }
    h2 { font-size: 1.75rem; }
    h3 { font-size: 1.5rem; }
    h4 { font-size: 1.25rem; }
    h5 { font-size: 1rem; }
    h6 { font-size: 0.875rem; }

    /* Paragraphs */
    p {
      margin-top: 0.3em;
      margin-bottom: 0.3em;
    }

    /* Links */
    a {
      text-decoration: none;
      background-color: transparent;
    }
    a:hover {
      text-decoration: underline;
    }

    /* Lists */
    ul, ol {
      margin-top: 0;
      margin-bottom: 1em;
    }
    ul ul, ul ol, ol ul, ol ol {
      margin-bottom: 0;
    }

    /* Code blocks */
    pre, code {
      font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
      font-size: 0.875em;
    }
    pre {
      margin-top: 0;
      margin-bottom: 1em;
      overflow: auto;
    }
    code {
      padding: 0.2em 0.4em;
      margin: 0;
      font-size: 85%;
      background-color: rgba(27,31,35,0.05);
      border-radius: 3px;
    }
    </style>
  
    <meta name="description" content="">
  </head>
<body>
  <header class="header">
    <div class="container">
      <div class="row">
        <!-- <div class="col-xs-2">
          <a href="../">
            <img class="img-fluid" src=../images/logo.png alt="ML4H: Machine Learning for Health">
          </a>
        </div> -->
        <div class="col-xs-10">
          <h1 class="title"><a href="../">ICML 2025 Workshop on Multi-modal Foundation Models and Large Language Models for Life Sciences</a></h1>
          <!-- <p class="text-muted">Workshop at NeurIPS 2020</p> -->
          <br />
          <ul class="list-inline" style="font-weight:600"> 
            <li class="list-inline-item"><a href="../index.html">Home</a></li>
            <li class="list-inline-item"><a href="../pages/call-for-participation.html">Call for Papers</a></li>
            <li class="list-inline-item"><a href="../pages/schedule.html">Schedule</a></li>
            <li class="list-inline-item"><a href="../pages/speakers.html">Speakers</a></li>
            <li class="list-inline-item"><a href="../pages/organizers.html">Organizers</a></li>
            <!-- <li class="list-inline-item"><a href="../pages/program-committee.html">Program Committee</a></li> -->
             <li class="list-inline-item"><a href="../pages/accepted-paper.html">Accepted Papers</a></li> 
          </ul>
        </div>
      </div>
    </div>
  </header>

<div class="main">
  <div class="container">
    <h1>Accepted Papers</h1>
    <p>
      <a href="https://openreview.net/group?id=ICML.cc/2025/Workshop/FM4LS#tab-accept"
         class="btn btn-primary"
         target="_blank"
         rel="noopener noreferrer">
        ðŸ“„ View Accepted Papers on OpenReview
      </a>
    </p>
    <hr>
    <article class="article">
      <div class="content" id="accepted-papers">
        <table class="table table-bordered table-hover">
          <tbody>
            <tr><td>Learning Diffusion Models with Flexible Representation Guidance (Contributed Talk)<br>
              <em>Chenyu Wang, Cai Zhou, Sharut Gupta, Zongyu Lin, Stefanie Jegelka, Stephen Bates, Tommi Jaakkola</em>
            </td></tr>

            <tr><td>NextGenPLM: A Novel Structure-Infused Foundational Protein Language Model for Antibody Discovery and Optimization (Contributed Talk)<br>
              <em>Abhinav Gupta, Ruijiang Li, Camila Leal, Madhumati Sevvana, Brian C Mackness, Ryan G Casner, Joseph D Batchelor, Anna Park, Michael Bailey, Lorenzo Kogler Anele, Sven Jager, Maria Wendt, Yves Fomekong Nanfack, Norbert Furtmann, Yu Qiu</em>
            </td></tr>

            <tr><td>Towards functional annotation with latent protein language model features (Contributed Talk)<br>
              <em>Jake Silberg, Elana Simon, James Zou</em>
            </td></tr>

            <tr></tr>
              
            <tr><td>3D-SBDD meets LLM: Towards FDA-Level Drug Design<br>
              <em>Bowen Gao, Yanwen Huang, Yiqiao Liu, Wenxuan Xie, Bowei He, Haichuan Tan, Wei-Ying Ma, Ya-Qin Zhang, Yanyan Lan</em>
            </td></tr>
            <tr><td>A Foundation Model for Mass Spectrometry Proteomics<br>
              <em>Justin Sanders, Melih Yilmaz, Jacob H. Russell, William E Fondrie, Wout Bittremieux, Nicholas M. Riley, Sewoong Oh, William Noble</em>
            </td></tr>
            <tr><td>A Genomic Language Model for Zero-Shot Prediction of Promoter Indel Effects<br>
              <em>Courtney A. Shearer, Felix Teufel, Rose Orenbuch, Christian J. Steinmetz, Daniel Ritter, Erik Xie, Artem Gazizov, Aviv Spinner, Jonathan Frazer, Mafalda Dias, Pascal Notin, Debora Susan Marks</em>
            </td></tr>
            <tr><td>A Look at the Isotropy of Pretrained Protein Language Models<br>
              <em>Sheikh Azizul Hakim, Kowshic Roy, M Saifur Rahman</em>
            </td></tr>
            <tr><td>A Multi-Modal Large Language Model for Free-Form, Open-Ended, and Interactive Prediction of Properties and Mechanisms of Candidate Drug Molecules<br>
              <em>Youwei Liang, Ruiyi Zhang, Zinnia Ma, Digvijay Singh, Yongce Li, Mingjia Huo, Chengzhan Gao, Hamidreza Rahmani, Satvik Bandi, Li Zhang, Robert Weinreb, Atul Malhotra, Danielle A. Grotjahn, Linda Awdishu, Trey Ideker, Michael K Gilson, Pengtao Xie</em>
            </td></tr>
            <tr><td>AIDO.Tissue: Spatial Cell-Guided Pretraining for Scalable Spatial Transcriptomics Foundation Model<br>
              <em>Jing Gong, Yixuan Wang, Nicholas Ho, Xingyi Cheng, Le Song, Eric P. Xing</em>
            </td></tr>
            <tr><td>Advancing Knotted Protein Design with ESM3: Guided Generation and Topological Insights<br>
              <em>Petr Simecek, Eva KlimentovÃ¡</em>
            </td></tr>
            <tr><td>AnnoDPO: Protein Functional Annotation Learning with Direct Preference Optimization<br>
              <em>Zixuan Jiang, Renjing Xu</em>
            </td></tr>
            <tr><td>ATOMICA: Learning Universal Representations of Intermolecular Interactions<br>
              <em>Ada Fang, Zaixi Zhang, Andrew Zhou, Marinka Zitnik</em>
            </td></tr>
            <tr><td>Benchmarking Vision-Language Contrastive Methods for Medical Representation Learning<br>
              <em>Shuvendu Roy, Yasaman Parhizkar, Franklin Ogidi, Vahid Reza Khazaie, Michael Colacci, Ali Etemad, Elham Dolatabadi, Arash Afkanpour</em>
            </td></tr>
            <tr><td>BioLangFusion: Multimodal Fusion of DNA, mRNA, and Protein Language Models<br>
              <em>Amina Mollaysa, Artem Moskalev, Pushpak Pati, Tommaso Mansi, Mangal Prakash, Rui Liao</em>
            </td></tr>
            <tr><td>Cell-Type-Aware Pooling for Robust Sample Classification in Single-Cell RNA-seq Data<br>
              <em>Soorin Yim, Kyungwook Lee, Dongyun Kim, Sungjoon Park, Doyeong Hwang, Soonyoung Lee, Amy Dunn, Daniel Gatti, Elissa Chesler, Kristen Oâ€™Connell, Kiyoung Kim</em>
            </td></tr>
            <tr><td>Challenges and Guidelines in Deep Generative Protein Design: Four Case Studies<br>
              <em>Tianyuan Zheng, Alessandro Rondina, Gos Micklem, Pietro Lio</em>
            </td></tr>
            <tr><td>Closing the gap between the biology and the clinic with a foundation model of immunology and inflammation<br>
              <em>Aziz FouchÃ©, Apolline Bruley, Matthew Corney, Pierre Marschall, Vincent Bouget, Julien Duquesne</em>
            </td></tr>
            <tr><td>Conditional Normalizing Flows for the Design of T Cell Therapies<br>
              <em>Samarth Kadaba, Alexander K Eapen, Kristin C. Y. Tsui, Kuan Pang, Theodore L Roth, Zinaida Good</em>
            </td></tr>
            <tr><td>DeepSeq: High-Throughput Single-Cell RNA Sequencing Data Labeling via Web Search-Augmented Agentic Generative AI Foundation Models<br>
              <em>Saleem Abdul Fattah Ahmed Al Dajani, Abel Sanchez, John R Williams</em>
            </td></tr>
            <tr><td>Describe Anything in Medical Images<br>
              <em>Xi Xiao, Yunbei Zhang, Thanh-Huy Nguyen, Ba-Thinh Lam, Janet Wang, Lin Zhao, Jihun Hamm, Tianyang Wang, Xingjian Li, Xiao Wang, Hao Xu, Tianming Liu, Min Xu</em>
            </td></tr>
            <tr><td>DisProtEdit: Exploring Disentangled Representations for Multi-Attribute Protein Editing<br>
              <em>Max Ku, Sun Sun, Hongyu Guo, Wenhu Chen</em>
            </td></tr>
            <tr><td>Enriched Instruction-Following Graph Alignment for Efficient Medical Vision-Language Models<br>
              <em>Duy Minh Ho Nguyen, Nghiem Tuong Diep, Trung Quoc Nguyen, Hoang-Bao Le, Tai Nguyen, Anh-Tien Nguyen, TrungTin Nguyen, Nhat Ho, Pengtao Xie, Roger Wattenhofer, Daniel Sonntag, James Zou, Mathias Niepert</em>
            </td></tr>
            <tr><td>Evaluating Multi-Modal Models for Enzyme-Reaction Retrieval<br>
              <em>Annika Viswesh, Jason Yang, Frances H. Arnold, Yisong Yue</em>
            </td></tr>
            <tr><td>From Vision to Graph Self-Supervised Learning in Digital Pathology<br>
              <em>Sevda Ã–ÄŸÃ¼t, Carlos Hurtado, CÃ©dric Vincent-Cuaz, Natalia Dubljevic, Vaishnavi Subramanian, Dorina Thanou, Pascal Frossard</em>
            </td></tr>
            <tr><td>GeneChat: A Multi-Modal Large Language Model for Gene Function Prediction<br>
              <em>Shashi Dhanasekar, Pengtao Xie, Akash Saranathan</em>
            </td></tr>
            <tr><td>H&Enium, Applying Foundation Models to Computational Pathology and Spatial Transcriptomics to Learn an Aligned Latent Space<br>
              <em>Marc Glettig, Tim Ehrensperger, Josephine Yates, Valentina Boeva</em>
            </td></tr>
            <tr><td>HPP-Voice: A Large-Scale Evaluation of Speech Embeddings for Multi-Phenotypic Classification<br>
              <em>David Krongauz, Hido Pinto, Sarah Kohn, Yanir Marmor, Eran Segal</em>
            </td></tr>
            <tr><td>Ibex: Pan-immunoglobulin structure prediction<br>
              <em>Frederic A Dreyer, Karolis Martinkus, Jan Ludwiczak, Brennan Abanades, Robert G Alberstein, Pranav Rao, Jae Hyeon Lee, Richard Bonneau, Andrew Martin Watkins, Franziska Seeger</em>
            </td></tr>
            <tr><td>Integrating Pathology Foundation Models and Spatial Transcriptomics for Cellular Decomposition from Histology Images<br>
              <em>Yutong Sun, Sichen Zhu, Peng Qiu</em>
            </td></tr>
            <tr><td>Joint Diffusion Sampling via Positive-Unlabeled Guidance for Multi-Modal Data<br>
              <em>Matt Raymond, Yilun Zhu, Jianxin Zhang, Angela Violi, Clayton Scott</em>
            </td></tr>
            <tr><td>KD-CPT: A Knowledge-Driven Cellular Phenotypic Transdifferentiation Model<br>
              <em>Lei Xin, Zhenglun Kong, Xiaoshuo Yan, Sijia Yan, Zeheng Wang, Wuzhe Fan, Hao Tang</em>
            </td></tr>
            <tr><td>Knowledge Graph-Augmented DNA Representation Learning<br>
              <em>Fengyu Cai, Erik Kubaczka, Shaobo Cui, Heinz Koeppl</em>
            </td></tr>
            <tr><td>Leveraging the Structure of Medical Data for Improved Representation Learning<br>
              <em>Andrea Agostini, Sonia Laguna, Alain Ryser, Samuel Ruiperez-Campillo, Moritz Vandenhirtz, Nicolas Deperrois, Farhad Nooralahzadeh, Michael Krauthammer, Thomas M. Sutter, Julia E Vogt</em>
            </td></tr>
            <tr><td>Ligand Iterative Sampling for Affinity Refinement and Drug Discovery (LISARDD)<br>
              <em>Valentin BADEA, Shyam Chandra, John Lin</em>
            </td></tr>
            <tr><td>MiST: Understanding the Role of Mid-Stage Scientific Training in Developing Chemical Reasoning Models<br>
              <em>Andres M Bran, Tong Xie, Shai Pranesh, Jeremy Goumaz, Xuan Vu Nguyen, David Ming Segura, Ruizhi Xu, Jeffrey Meng, Dongzhan Zhou, Wenjie Zhang, Philippe Schwaller</em>
            </td></tr>
            <tr><td>Molecularly informed analysis of histopathology images using natural language<br>
              <em>Moritz Schaefer, Kalin Nonchev, Animesh Awasthi, Jake Burton, Viktor Koelzer, Gunnar Ratsch, Christoph Bock</em>
            </td></tr>
            <tr><td>Multi-Modal Interpretable Graph for Competing Risk Prediction with Electronic Health Records<br>
              <em>Munib Mesinovic, Peter Watkinson, Tingting Zhu</em>
            </td></tr>
            <tr><td>Multi-Modal Large Language Model Enables Protein Function Prediction<br>
              <em>Han Guo, Mingjia Huo, Xingyi Cheng, Digvijay Singh, Hamidreza Rahmani, Shen Li, Philipp Gerlof, Trey Ideker, Danielle Grotjahn, Elizabeth Villa, Le Song, Pengtao Xie</em>
            </td></tr>
            <tr><td>Multi-Modal Medical Image Augmentation for Controlled Heterogeneity and Fair Outcomes<br>
              <em>Soo Yong Kim, Seunghyeok Hong</em>
            </td></tr>
            <tr><td>Multi-Objective-Guided Discrete Flow Matching for Controllable Biological Sequence Design<br>
              <em>Tong Chen, Yinuo Zhang, Sophia Tang, Pranam Chatterjee</em>
            </td></tr>
            <tr><td>Multi-Objective-Guided Generative Design of mRNA with Therapeutic Properties<br>
              <em>Sawan Patel, Sophia Tang, Yinuo Zhang, Pranam Chatterjee, Sherwood Yao</em>
            </td></tr>
            <tr><td>Multimillion cell self-supervised representation learning enables organ-scale tissue niche discovery<br>
              <em>Alex Jihun Lee, Alma Dubuc, Michael Kunst, Shenqin Yao, Nicholas Lusk, Lydia Ng, Hongkui Zeng, Bosiljka Tasic, Reza Abbasi-Asl</em>
            </td></tr>
            <tr><td>Multimodal Benchmarking of Foundation Model Representations for Cellular Perturbation Response Prediction<br>
              <em>Euxhen Hasanaj, Elijah Cole, Shahin Mohammadi, Sohan Addagudi, Xingyi Zhang, Le Song, Eric P. Xing</em>
            </td></tr>
            <tr><td>Multimodal Modeling of CRISPR-Cas12 Activity Using Foundation Models and Chromatin Accessibility Data<br>
              <em>Azim Dehghani Amirabad, Yanfei Zhang, Artem Moskalev, Sowmya Rajesh, Tommaso Mansi, Shuwei Li, Mangal Prakash, Rui Liao</em>
            </td></tr>
            <tr><td>Multimodal Protein Language Models for Flexibility Prediction and Loop Design<br>
              <em>Tyler Verhaar, Samuel W. K. Wong</em>
            </td></tr>
            <tr><td>PM1: A Foundation Model Fusing Genotype, Phenotype, and Image for Precision Medicine<br>
              <em>Margarita Geleta, Christophe Thomassin, MarÃ§al Comajoan Cara, David Bonet, Aina Luis Vidal, Benet Oriol Sabat, Daniel Mas Montserrat, Alexander G. Ioannidis</em>
            </td></tr>
            <tr><td>Promoter Sequence Generation using Homology Prompting<br>
              <em>Erik Xie, Courtney A. Shearer, Ruben Weitzman, Pascal Notin, Debora Susan Marks</em>
            </td></tr>
            <tr><td>ProteinAligner: A Tri-Modal Contrastive Learning Framework for Protein Representation Learning<br>
              <em>Li Zhang, Han Guo, Leah Schaffer, Young Su Ko, Digvijay Singh, Danielle Grotjahn, Elizabeth Villa, Michael K Gilson, Wei Wang, Trey Ideker, Eric P. Xing, Pengtao Xie</em>
            </td></tr>
            <tr><td>ProteinGPT: Multimodal LLM for Protein Property Prediction and Structure Understanding<br>
              <em>Yijia Xiao, Edward Sun, Yiqiao Jin, Qifan Wang, Wei Wang</em>
            </td></tr>
            <tr><td>Rapid and Reproducible Multimodal Biological Foundation Model Development with AIDO.ModelGenerator<br>
              <em>Caleb Ellington, Dian Li, Shuxian Zou, Elijah Cole, Ning Sun, Sohan Addagudi, Le Song, Eric P. Xing</em>
            </td></tr>
            <tr><td>RepoLLM: A Multi-modal Foundation Model for Drug Repurposing via Alignment of Molecules, EHRs, and Knowledge Graphs<br>
              <em>Zichao Li, Zong Ke</em>
            </td></tr>
            <tr><td>Retrieval Augmented Protein Language Models for Protein Structure Prediction<br>
              <em>Pan Li, Xingyi Cheng, Le Song, Eric P. Xing</em>
            </td></tr>
            <tr><td>Robust Multi-Omics Integration from Incomplete Modalities Significantly Improves Prediction of Alzheimerâ€™s Disease<br>
              <em>Sungjoon Park, Kyungwook Lee, Soorin Yim, Doyeong Hwang, Dongyun Kim, Soonyoung Lee, Amy Dunn, Daniel Gatti, Elissa Chesler, Kristen O'Connell, Kiyoung Kim</em>
            </td></tr>
            <tr><td>Scaling up measurement noise scaling laws<br>
              <em>Igor Sadalski, Dan Raviv, Jonathan S Rosenfeld, Allon M Klein, Gokul Gowri</em>
            </td></tr>
            <tr><td>Segmentation Helps Understanding: Mask-Infused Vision-Language Pre-training for 3D Medical Images<br>
              <em>Yuqi Hu, Xufang Luo, Zilong Wang, Dongsheng Li, Lili Qiu</em>
            </td></tr>
            <tr><td>Self-Supervised Representation Learning for Microbiome Improves Downstream Prediction in Data-Limited Settings and Cross-Cohort Generalizability<br>
              <em>Liron Zahavi, Zachary Levine, Eran Segal</em>
            </td></tr>
            <tr><td>SHIVER: Somatic Hypermutation Informed Vocabulary Encoder Representations<br>
              <em>Chiho Im, Artem Mikelov, Ryan Zhao, Anshul Kundaje, Scott D. Boyd</em>
            </td></tr>
            <tr><td>SOAPIA: Siamese-Guided Generation of Off Target-Avoiding Protein Interactions with High Target Affinity<br>
              <em>Sophia Vincoff, Oscar Davis, Ismail Ilkan Ceylan, Alexander Tong, Joey Bose, Pranam Chatterjee</em>
            </td></tr>
            <tr><td>Stabilizing protein fitness predictors via the PCS framework<br>
              <em>Omer Ronen, Alex Y. Zhao, Ron Boger, Chengzhong Ye, Bin Yu</em>
            </td></tr>
            <tr><td>SYNAPSE: A Multi-Modal Framework for Interpretable Neural Decoding Using Vision-Language Foundation Models<br>
              <em>Edward Ju, Adarsh Kumarappan, Shrujana S Kunnam, Raaghav Malik, Dhruv Sheth</em>
            </td></tr>
            <tr><td>TICA-Based Free Energy Matching for Machine-Learned Molecular Dynamics<br>
              <em>Alexander Aghili, Andy Bruce, Daniel Sabo, Razvan Marinescu</em>
            </td></tr>
            <tr><td>Temporal Representation Learning for Ultrasound Analysis using Masked Modeling<br>
              <em>Yves Dominic Stebler, Thomas M. Sutter, Ece Ozkan, Julia E Vogt</em>
            </td></tr>
            <tr><td>Towards foundation models that learn across biological scales<br>
              <em>Jeremie Kalfon, Laura Cantini, Gabriel PeyrÃ©</em>
            </td></tr>
            <tr><td>Towards Molecular Conformer Generation with Language Models<br>
              <em>Menua Bedrosian, Hrant Khachatrian</em>
            </td></tr>
            <tr><td>Transfer Learning of Condition-Specific Perturbation in Gene Interactions: Towards Multi-modal Foundational Modeling of Drug Response<br>
              <em>Dongmin Bang, Bonil Koo, Sun Kim</em>
            </td></tr>
            <tr><td>Transferring Cell-level Drug Response to Patient via Tumor Heterogeneity-Aware Alignment and Gene-level Foundational Models<br>
              <em>Inyoung Sung, Dongmin Bang, Sun Kim, Sangseon Lee</em>
            </td></tr>
            <tr><td>TRIDENT: Tri-Modal Molecular Representation Learning with Taxonomic Annotations and Local Correspondence<br>
              <em>Feng Jiang, Mangal Prakash, Hehuan Ma, Jianyuan Deng, Yuzhi Guo, Amina Mollaysa, Tommaso Mansi, Rui Liao, Junzhou Huang</em>
            </td></tr>
            <tr><td>Unified Representation of Genomic and Biomedical Concepts through Multi-Task, Multi-Source Contrastive Learning<br>
              <em>Hongyi Yuan, Suqi Liu, Zongxin Yang, Kelly Cho, Katherine P. Liao, Alexandre Pereira, Tianxi Cai</em>
            </td></tr>
            <tr><td>Uncertainty-Aware Discrete Diffusion Improves Protein Design<br>
              <em>Sazan Mahbub, Christoph Feinauer, Caleb Ellington, Le Song, Eric P. Xing</em>
            </td></tr>
          </tbody>

        </table>
      </div>
    </article>
  </div>
</div>

</body>

</html>
